components/brave_wallet_ui/page/screens/backup-wallet/backup-enter-password/backup-enter-password.tsx

```diff
@@ -93,6 +93,11 @@ export const BackupEnterPassword: React.FC = () => {
     setPassword(value)
   }
 
+  const goBackUrl =
+    history.action === 'POP'
```

Comment:
bit: looks like `Action.POP` is available for import: https://github.com/remix-run/history/blob/3e9dab413f4eda8d6bce565388c5ddb7aeff9f7e/packages/history/index.ts#L6